test/minitest_helper.rb

```diff
@@ -24,4 +24,4 @@
 require 'minitest/autorun'
 require 'support/database'
 require 'support/db_test_case'
-
+require 'support/article_entity'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.